mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1920,30 +1983,52 @@
     <string name="login_menu_edit_button">Edit</string>
     <!-- Message in delete confirmation dialog for logins -->
     <string name="login_deletion_confirmation">Are you sure you want to delete this login?</string>
+    <!-- Message in delete confirmation dialog for password -->
+    <string name="login_deletion_confirmation_2" tools:ignore="UnusedResources">Are you sure you want to delete this password?</string>
     <!-- Positive action of a dialog asking to delete  -->
     <string name="dialog_delete_positive">Delete</string>
     <!-- Negative action of a dialog asking to delete login -->
     <string name="dialog_delete_negative">Cancel</string>
     <!--  The saved login options menu description. -->
     <string name="login_options_menu">Login options</string>
+    <!--  The saved password options menu description. -->
+    <string name="login_options_menu_2" tools:ignore="UnusedResources">Password options</string>
     <!--  The editable text field for a login's web address. -->
     <string name="saved_login_hostname_description">The editable text field for the web address of the login.</string>
+    <!--  The editable text field for a password's website address. -->
+    <string name="saved_login_hostname_description_2" tools:ignore="UnusedResources">The editable text field for the website address of the password.</string>
     <!--  The editable text field for a login's username. -->
     <string name="saved_login_username_description">The editable text field for the username of the login.</string>
+    <!--  The editable text field for a password's username. -->
+    <string name="saved_login_username_description_2" tools:ignore="UnusedResources">The editable text field for the username of the password.</string>
```

Comment:
"login" was replaced with "password", but this sentence doesn't make a lot of sense now.

Reply:
@flodolo - Would "The editable text field for the username associated with the password." be more palatable?

Reply:
That works, but maybe just "for the username" is enough?

Reply:
My mind went there, too, Flod.

@sarah541 - Let's do: The editable text field for the username. 

Following this same pattern, there's another string to update.

saved_login_hostname_description = The editable text field for the website address.